src/renderer/entities/wallet/model/wallet-model.ts

```diff
@@ -220,8 +241,49 @@ sample({
   target: $wallets,
 });
 
+sample({
+  clock: walletHidden,
+  fn: (wallet) => wallet,
+  target: hideWalletFx,
```

Comment:
As I remember we can do just this (before effector v23, there was a `forward` operator doing exactly the same)
```suggestion
sample({
  clock: walletHidden,
  target: hideWalletFx,
```